pkgs/development/libraries/mesa/default.nix

```diff
@@ -59,29 +60,25 @@ let
   driDrivers =
     (if dri_ == null
       then optionals (elem "drm" eglPlatforms) defaultDriDrivers
-      else dri_) ++ lib.optional stdenv.isLinux "swrast";
```

Comment:
I'm not sure where/when to build gallium vs dri `swrast`... Also, do we not want llvmpipe/softpipe in here anywhere?